projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -71,7 +71,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net451" Version="1.0.0" PrivateAssets="All" />
```

Comment:
What failure were you seeing? You shouldn't need to directly reference those sub packages. That should be correctly handled by referencing the `Microsoft.NETFramework.ReferenceAssemblies` package.

Reply:
On Linux and OS X builds -

https://gist.github.com/lukebakken/30d05fc5f03ee9ff47527e5d09864f26

This is the `Vagrantfile` that can bring up a Debian 10 VM with the same packages as our CI build - https://gist.github.com/lukebakken/676bb6d75e30aa5a0bc533f9af87a336

I have no idea what's up, just that these changes fix the builds! 🤷‍♂

Reply:
That doesn't really make sense though, because those packages would already be referenced with the original reference.

You would need to add Microsoft.NETFramework.ReferenceAssemblies to the WinRT project, but shouldn't need to reference a specific one directly like that.

Reply:
I will give the WinRT suggestion a try. That's probably it.

Reply:
https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/710/commits/e7578bf8efb412a6c5d6c3338040a274dfcf3796 works. The reference is required in all of those `.csproj` files.

Reply:
Thanks for double-checking.

Reply:
If you're still having a problem with that change, post the build log failure, and I'll see if I can figure it out.

Ultimately I guess it does't matter, but it always bothers me when there's something weird going on without an explanation why. 😄

Reply:
Ah, should have refreshed before posting. Glad to see that solved it!

Reply:
> it always bothers me when there's something weird going on without an explanation why.

Me too, so thanks for checking. I don't have the time to dig into package / assembly / nuget issues and it's been long enough that I've been out of the .NET world that I'm quite rusty.

Reply:
Things have also changed pretty dramatically with the introduction of PackageReferences and the new project system.